src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -116,10 +116,16 @@ public static function castType(\ReflectionType $c, array $a, Stub $stub, bool $
 
     public static function castAttribute(\ReflectionAttribute $c, array $a, Stub $stub, bool $isNested)
     {
-        self::addMap($a, $c, [
+        $map = [
             'name' => 'getName',
             'arguments' => 'getArguments',
-        ]);
+        ];
+
+        if (\PHP_VERSION_ID >= 80400) {
+            unset($map['name']);
+        }
```

Comment:
Unsetting _after_ is needed to keep `name` in first position of the array if it's used